tensorflow_quantum/python/differentiators/gradient_test.py

```diff
@@ -220,5 +236,97 @@ def test_analytic_value_with_simple_circuit(self, differentiator, op):
         self.assertAllClose(ground_truth_grads, grads, rtol=1e-2, atol=1e-2)
 
 
+class SampledGradientCorrectnessTest(tf.test.TestCase, parameterized.TestCase):
+    """Test approximate correctness to analytical methods."""
+
+    @parameterized.parameters(
+        list(
+            util.kwargs_cartesian_product(
+                **{
+                    'differentiator': SAMPLED_DIFFS,
+                    'op': SAMPLED_OPS,
+                    'num_samples': [10000]
+                })))
+    def test_sampled_value_with_simple_circuit(self, differentiator, op,
+                                               num_samples):
+        """Test the value of sampled differentiator with simple circuit."""
+        # Get an expectation op, with this differentiator attached.
+        differentiator.refresh()
+        op = differentiator.generate_differentiable_op(sampled_op=op)
+        qubit = cirq.GridQubit(0, 0)
+        circuit = util.convert_to_tensor(
+            [cirq.Circuit(cirq.X(qubit)**sympy.Symbol('alpha'))])
+        psums = util.convert_to_tensor([[cirq.Z(qubit)]])
+        symbol_values_array = np.array([[0.123]], dtype=np.float32)
+        # Calculate tfq gradient.
+        symbol_values_tensor = tf.convert_to_tensor(symbol_values_array)
+        with tf.GradientTape() as g:
+            g.watch(symbol_values_tensor)
+            expectations = op(circuit, tf.convert_to_tensor(['alpha']),
+                              symbol_values_tensor, psums,
+                              tf.convert_to_tensor([[num_samples]]))
+        grads = g.gradient(expectations, symbol_values_tensor)
+        ground_truth_grads = np.array([[-1.1839752]])
+        self.assertAllClose(ground_truth_grads, grads, rtol=0.2, atol=0.2)
```

Comment:
These tolerances seem a bit too large to me, could the `rtol` be removed and the `atol` be tightened up to something like `0.02`?